### PR TITLE
Variables and functions test update

### PR DESCRIPTION
**Pull request by sofiiako**
Updated the task (changed the solution, edited placeholder)

### CODE REVIEW

**Review by Geravant — CHANGES_REQUESTED** [review]
It is all good!
Would we move the if lesson before this as well?
We can also rename it into smth like "Veriables, functions and if test", though it might be a bit long

**Comment by sofiiako** [comment]
@Geravant requested changes here: https://github.com/jetbrains-academy/rustlings-course/commit/26802f207186d5f341e8b9020cefe4262f0b594b

File: Common Programming Concepts/section-info.yaml [hunk 1]
```diff
@@ -2,5 +2,5 @@ content:
 - Variables
 - Data Types
 - Functions
-- Variables and Functions Test
+- "Variables, Functions and If Test"
 - If
```
Comment (Geravant) [inline Common Programming Concepts/section-info.yaml:6]: Should we move the if lesson before the "Variables, Function and If test"?

File: Common Programming Concepts/section-info.yaml [hunk 2]
```diff
@@ -2,5 +2,5 @@ content:
 - Variables
 - Data Types
 - Functions
-- Variables and Functions Test
+- "Variables, Functions and If Test"
```
Comment (Geravant) [inline Common Programming Concepts/section-info.yaml:5]: Does this attach the lesson correctly? I get an error with the quotes, but it seems to work fine without them/

**Review by Geravant — CHANGES_REQUESTED** [review]

**Review by Geravant — APPROVED** [review]

**Pull request merged**